test/browser_tests/spec_suites/bureau-structure.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+var TheBureauStructurePage =
+require( '../page_objects/page_bureau-structure.js' );
+
+describe( 'The Bureau Structure Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new TheBureauStructurePage();
+    page.get();
+  } );
+
+
+  it( 'should properly load in a browser',
+    function() {
+      expect( page.pageTitle() ).toBe( 'Bureau Structure' );
+    }
+  );
+
+  it( 'should have a side nav',
+    function() {
+      expect( page.sideNav.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Director\'s image',
+    function() {
+      expect( page.directorImage.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Director\'s name',
+    function() {
+      expect( page.directorName.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Director\'s title',
+    function() {
+      expect( page.directorTitle.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include two org chart branches',
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/8555b964c486de0b03084f0e58e3f97f44f7727a